docs/langref/src/legacy_syntax.md

```diff
@@ -1,14 +1,21 @@
-## Legacy syntax
+# Legacy Syntax
 
-To keep compatibility with previous version of Slint, the old syntax that declared component with `:=` is still valid
+To keep compatibility with earlier version of Slint, the old syntax that declared
+components and named structs with `:=` is still valid:
 
 ```slint,no-preview
 export MyApp := Window {
     //...
 }
 ```
 
-Element position and property lookup is different in the new syntax.
-In the new syntax, only property declared within the component are in scope.
-In the previous syntax, all properties of bases of `self` and `root` were also in scope.
-In the new syntax, elements are centered by default and the constraints are applied to the parent.
+This syntax change also effects property lookup rules and default element placement.
+
+In components defined in the new syntax, only properties declared within the
+component are in scope. By default parent elements render their children centered
+and will apply all layout constraints.
+
+In components defined using old systax, all properties of bases of `self` and
+`root` were in scope in addition to all properties defined inside the component
+itself. Elements are placed at position `x: 0` and `y: 0` and their constraints
+aren't applied.
```

Comment:
I realise that it's odd that the _new_ syntax is explained within the _legacy_ section. If anything, this kind of explanation belongs into a "Migration" section IMO.

Reply:
It depends on wethere the legacy syntax is still supported in 1.0 or not.
If it is still supported, i think we need both the details in a reference, and in a migration section.
If we don't support it anymore, it can be only in a migration section.